democracy/models/gdpr_data_serialization_mixin.py

```diff
@@ -0,0 +1,14 @@
+from kerrokantasi.utils import get_current_request
+
+
+class FileFieldUrlSerializerMixin:
+    field_to_use_as_url_field = None  # Override in subclass.
+
+    @property
+    def url(self):
+        field = getattr(self, self.field_to_use_as_url_field, None)
```

Comment:
Since this is initialized with `None` already you can actually just do the following here:
```suggestion
        field =  self.field_to_use_as_url_field
```